nss-day-cohort-43/nuttyshell-sons-of-witches
# Bs adding functionality to add friend button

# Description
Added functionality to prepend usernames with an @ symbol when messages are saved and an Add Friend button now appears whenever a username beside a saved messaged is clicked.

Fixes # (issue)
## Type of change
Please delete options that are not relevant.

# Testing Instructions
Please ensure an @ symbol appears before all usernames associated with submitted messages and an Add Friend button appears whenever you click the username beside a saved message.

# Checklist:
